src/main/java/com/iabtcf/decoder/ConsentParser.java

```diff
@@ -0,0 +1,311 @@
+
+    /**
+     * The vendor identifiers that have consent to process this users personal data. The vendor
+     * identifiers are published in the GVL.
+     * 
+     * @since 1.0
+     * @return the vendor identifiers.
+     */
+    IntIterable getVendorConsent();
+
+    /**
+     * Default consent for VendorIds not covered by a RangeEntry. VendorIds covered by a RangeEntry
+     * have a consent value the opposite of DefaultConsent.
+     * 
+     * This field in not used by Transparency and Consent String v2.0 specifications and always
+     * returns false.
+     * 
+     * @since 1.0
+     * @return all vendors that have consent to process this users personal data
+     */
+    boolean getDefaultVendorConsent();
+
+    /**
+     * From the corresponding field in the GVL that was used for obtaining consent. A new policy
+     * version invalidates existing strings and requires CMPs to re-establish transparency and
+     * consent from users.
+     *
+     * @since 2.0
+     * @return version of policy used within GVL
+     */
+    byte getTcfPolicyVersion();
+
+    /**
+     * Whether the signals encoded in this TC String were from service-specific storage versus
+     * global consesu.org shared storage.
+     *
+     * @since 2.0
+     * @return if signals are service-specific or global
+     */
+    boolean getIsServiceSpecific();
+
+    /**
+     * Setting this to field to true means that a publisher-run CMP – that is still IAB Europe
+     * registered – is using customized Stack descriptions and not the standard stack descriptions
+     * defined in the Policies (Appendix A section E). A CMP that services multiple publishers sets
+     * this value to false.
+     *
+     * @since 2.0
+     * @return true if if the CMP used non-IAB standard stacks during consent gathering; false
+     *         otherwise.
+     */
+    boolean getUseNonStandardStacks();
+
+    /**
+     * The TCF Policies designates certain Features as "special" which means a CMP must afford the
+     * user a means to opt in to their use. These "Special Features" are published and numerically
+     * identified in the Global Vendor List separately from normal Features.
+     *
+     * @since 2.0
+     * @return the Special Features the Vendor may utilize when performing some declared Purposes
+     *         processing.
+     */
+    IntIterable getSpecialFeatureOptIns();
+
+    /**
+     * The Purpose’s transparency requirements are met for each Purpose on the legal basis of
+     * legitimate interest and the user has not exercised their "Right to Object" to that Purpose.
+     * 
+     * By default or if the user has exercised their "Right to Object" to a Purpose, the
+     * corresponding identifier for that Purpose is set to false.
+     * 
+     * @since 2.0
+     * @return The purpose identifiers for which the legal basis of legitimate interest are
+     *         established.
+     */
+    IntIterable getPurposesLITransparency();
```

Comment:
Other methods do not include the word Transparency (getVendorLegitimateInterest()). We should standardize on one naming scheme. I vote for spelling out LegitimateInterest.

Reply:
The names are taken directly from the spec, which is desirable. I don't think we want to invent new names.